docs/src/content/docs/ko/guides/pages.mdx

```diff
@@ -0,0 +1,173 @@
+
+import FileTree from '~/components/file-tree.astro';
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+      - reference/
+        - faq.md
+
+</FileTree>
+
+### 타입 안정성을 갖춘 프론트매터
+
+모든 Starlight 페이지는 페이지 표시 방법을 제어하기 위해 사용자 정의 가능한 [공통 프론트매터 속성 세트](/ko/reference/frontmatter/)를 공유합니다.
+
+```md
+---
+title: 안녕하세요!
+description: 이것은 내 Starlight 기반 사이트의 페이지입니다.
+---
+```
+
+중요한 사항을 잊어버리면 Starlight가 알려드립니다.
+
+## 사용자 정의 페이지
+
+고급 사용 사례의 경우 `src/pages/` 디렉터리를 생성하여 사용자 정의 페이지를 추가할 수 있습니다.
+`src/pages/` 디렉터리는 [Astro의 파일 기반 라우팅](https://docs.astro.build/ko/basics/astro-pages/#파일-기반-라우팅)을 사용하며 다른 페이지 형식에서 `.astro` 파일에 대한 지원을 포함합니다.
+이는 완전히 사용자 정의된 레이아웃으로 페이지를 작성하거나 대체 데이터 소스에서 페이지를 생성해야 하는 경우에 유용합니다.
+
+예를 들어, 이 프로젝트는 `src/content/docs/`의 Markdown 콘텐츠를 `src/pages/`의 Astro 및 HTML 경로와 혼합합니다.
+
+<FileTree>
+
+- src/
+  - content/
+    - docs/
+      - hello-world.md
+  - pages/
+    - custom.astro
+    - archived.html
+
+</FileTree>
+
+[Astro 문서의 "페이지" 가이드](https://docs.astro.build/ko/basics/astro-pages/)에서 자세한 내용을 읽어보세요.
+
+### 사용자 정의 페이지에서 Starlight 디자인 사용
+
+사용자 정의 페이지에서 Starlight 레이아웃을 사용하려면 `<StarlightPage />` 컴포넌트로 페이지 콘텐츠를 감싸세요.
+이는 콘텐츠를 동적으로 생성하지만 여전히 Starlight의 디자인을 사용하려는 경우 유용할 수 있습니다.
+
+```astro
+---
+// src/pages/custom-page/example.astro
+import StarlightPage from '@astrojs/starlight/components/StarlightPage.astro';
+import CustomComponent from './CustomComponent.astro';
+---
+
+<StarlightPage frontmatter={{ title: '사용자 정의 페이지' }}>
+	<p>이것은 사용자 정의 컴포넌트가 있는 사용자 정의 페이지입니다.</p>
+	<CustomComponent />
+</StarlightPage>
+```
+
+#### Props
+
+`<StarlightPage />` 컴포넌트는 다음 props를 허용합니다.
+
+##### `frontmatter` (필수)
+
+**타입:** `StarlightPageFrontmatter`
+
+Markdown 페이지의 프론트매터와 유사하게 이 페이지에 대한 [프론트매터 속성](/ko/reference/frontmatter/)을 설정합니다.
+[`title`](/ko/reference/frontmatter/#title-필수) 속성은 필수이며 다른 모든 속성은 선택 사항입니다.
+
+다음 속성은 Markdown 프론트매터와 다릅니다.
+
+- [`slug`](/ko/reference/frontmatter/#slug) 속성은 지원되지 않으며 사용자 정의 페이지의 URL을 기반으로 자동 설정됩니다.
+- [`editUrl`](/ko/reference/frontmatter/#editurl) 옵션에는 편집 링크를 표시하기 위한 URL이 필요합니다.
+- [자동 생성된 링크 그룹](/ko/reference/configuration/#sidebar)에서 페이지가 표시되는 방식을 사용자 정의하기 위한 [`sidebar`](/ko/reference/frontmatter/#sidebar) 프론트매터 속성을 사용할 수 없습니다. `<StarlightPage />` 컴포넌트를 사용하는 페이지는 컬렉션의 일부가 아니며 자동 생성된 사이드바 그룹에 추가될 수 없습니다.
+
+##### `sidebar`
+
+**타입:** `SidebarEntry[]`  
+**기본값:** [전역 `sidebar` 구성](/ko/reference/configuration/#sidebar)을 기반으로 생성된 사이드바
+
+이 페이지에 대한 사용자 정의 사이트 탐색 사이드바를 제공합니다.
+설정하지 않으면 페이지에서 기본 전역 사이드바를 사용합니다.
+
+예를 들어, 다음 페이지는 홈페이지 링크와 다양한 별자리에 대한 링크 그룹으로 기본 사이드바를 재정의합니다.
+사이드바의 현재 페이지는 `isCurrent` 속성을 사용하여 설정되었으며 선택적인 `badge`가 링크 항목에 추가되었습니다.
+
+```astro {3-13}
+<StarlightPage
+	frontmatter={{ title: 'Orion' }}
+	sidebar={[
+		{ label: '홈', href: '/' },
+		{
+			label: '별자리',
+			items: [
+				{ label: '안드로메다', href: '/andromeda/' },
+				{ label: '오리온', href: '/orion/', isCurrent: true },
+				{ label: '작은곰자리', href: '/ursa-minor/', badge: 'Stub' },
```

Comment:
Is "Stub" not being translated on purpose? I know in some languages this was translated but some others left it as-is. I'll let you decide as you know more than me but just mentioning it in case this was skipped by accident.

Reply:
I intentionally left it because I couldn't find an appropriate translation.